source/BoardingPanel.cpp

```diff
@@ -536,17 +536,19 @@ void BoardingPanel::DoKeyboardNavigation(const SDL_Keycode key)
 
 // Constructor (commodity cargo).
 BoardingPanel::Plunder::Plunder(const string &commodity, int count, int unitValue)
-	: name(commodity), outfit(nullptr), count(count), unitValue(unitValue)
+	: name(commodity), outfit(nullptr), count(count), unitValue(unitValue), knownToPlayer(true)
 {
 	UpdateStrings();
 }
 
 
 
 // Constructor (outfit installed in the victim ship or transported as cargo).
-BoardingPanel::Plunder::Plunder(const Outfit *outfit, int count)
-	: name(outfit->DisplayName()), outfit(outfit), count(count),
-	unitValue(outfit->Cost() * (outfit->Get("installable") < 0. ? 1 : Depreciation::Full()))
+BoardingPanel::Plunder::Plunder(const Outfit *outfit, int count, bool outfitIsKnown)
+	: name(outfitIsKnown ? outfit->DisplayName() : "Unknown Outfit"), outfit(outfit), count(count),
```

Comment:
you forgot to check for count being different than 1 and having outfit*s*

Reply:
something like that?
name(outfitIsKnown ? outfit->DisplayName() : "Unknown Outfit" + (count == 1) ? "" : "s"),
?

Reply:
yeah
maybe just make it name = smth in the {} of the constructor honestly or it'd get a bit too big?

Reply:
I need to first get it to work, then I'll see if it needs to more into the body of the constructor.

Reply:
I wonder, the original code doesn't uses plurals, why this change is needed?

Reply:
oh it doesnt? my bad then dont touch it nvm